app/locales/de.json

```diff
@@ -35,11 +35,12 @@
       "id": "section_1",
       "title": "I. Basisdaten",
       "disclaimer": "Innovationsausgaben sowie Ausgaben für Forschung und Entwicklung: Tendenzen aus den Vorjahren setzen sich fort.",
-      "sectionDescription": "Hier in unserem Chart der Basisdaten können Sie sich die Daten nach Belieben zusammenstellen. Wählen sie Größen wie <strong>Branchen, Beschäftigungsgröße</strong> oder <strong>Technologieklasse</strong> und vergleichen verschiedene Kennzahlen und Ausgabengrößen. <br><br>Um die Daten besser in Kontext setzen zu können haben Sie die Wahl die Daten aus Berlin mit dem deckungsgleich aufgebauten Datensatz über bundesweite Innovationsunternehmen zu vergleichen.",
-      "toggleText": "Zusammensetzung der FuE-Ausgaben in ",
-      "chart-ids": [
-        "base_chart_ber",
-        "base_chart_ger"
+      "charts": [
+        {
+          "id": "base_chart",
+          "chartDescription": "Hier in unserem Chart der Basisdaten können Sie sich die Daten nach Belieben zusammenstellen. Wählen sie Größen wie <strong>Branchen, Beschäftigungsgröße</strong> oder <strong>Technologieklasse</strong> und vergleichen verschiedene Kennzahlen und Ausgabengrößen. <br><br>Um die Daten besser in Kontext setzen zu können haben Sie die Wahl die Daten aus Berlin mit dem deckungsgleich aufgebauten Datensatz über bundesweite Innovationsunternehmen zu vergleichen.",
```

Comment:
Super tiny thing: Since we’re already in the "chart" context, I would just use `description` for this field name. Otherwise, we’d have `chart.chartDescription`, which is a bit redundant or even confusing.

Reply:
jep. agreed